### DISCUSSION TITLE
Bigtreetech PITFT50

### DISCUSSION THREAD

**Discussion by D20180** [KlipperScreen/KlipperScreen#32] [General]
Will this run on a Bigtreetech PITFT50?
It doesn't have a HDMI interface just the DFI ribbon connector for the Pi.

**Comment by jordanruthe** [comment]
Yes it will.

**Reply by D20180** [reply]
Thanks! 
Sounds like a perfect match.

**Comment by simonvez** [comment]
it definately is working perfect on mine
